### ISSUE TITLE
Problems with cyrillic names of the folders

### ISSUE THREAD

**Issue by AndrewTCEC** [fohristiwhirl/nibbler#132]
Basically if any folder has it's name in Cyrillic, Nibbler is going bonkers. Can't load the nets and TBs, only engine load itself works fine.
Any ideas?

**Comment by fohristiwhirl** [comment]
Can you post an example of such a path?

**Comment by fohristiwhirl** [comment]
It's possible it's related to <https://github.com/LeelaChessZero/lc0/issues/1474>

**Comment by AndrewTCEC** [comment]
> It's possible it's related to [LeelaChessZero/lc0#1474](https://github.com/LeelaChessZero/lc0/issues/1474)

Possible, but the engine itself loads properly while nets and TBs don't. Here are some pics. Engine itself is located in the same "Engines" directory but inn a different folder and loads well and clear.
![image](https://user-images.githubusercontent.com/77082125/110852823-e492ed80-82c3-11eb-8e2b-63835b9ed2fb.png)
![image](https://user-images.githubusercontent.com/77082125/110853016-23c13e80-82c4-11eb-9552-65a5a5f3de9e.png)

**Comment by fohristiwhirl** [comment]
> the engine itself loads properly

Is that so when you use setoption via console?

**Comment by AndrewTCEC** [comment]
This is... how?

**Comment by fohristiwhirl** [comment]
That is, try running lc0.exe directly and doing:

```
setoption name WeightsFile value PATH
```

But replace PATH with the full absolute path (the bottom blue text in your screenshot).

I would predict Lc0 will emit the same error message, meaning it's an Lc0 problem.

**Comment by AndrewTCEC** [comment]
Okay

**Comment by fohristiwhirl** [comment]
You might need to also type:

```
go infinite
```

Afterwards, just to get it to actually load.

**Comment by AndrewTCEC** [comment]
Yeah, tried it for the net, no Cyrillic  - fine, yes Cyrillic - not fine. This one is an Lc0 issue. But what about TBs?

**Comment by fohristiwhirl** [comment]
I guess it's the same root cause in Lc0's code.

**Comment by AndrewTCEC** [comment]
Okay. Will wait for a fix patiently.

**Comment by rooklift** [comment]
By the way, you might want to give them a nudge about this.

**Comment by AndrewTCEC** [comment]
An update on the problem: in Nibbler 2.1.4 the warning is now ensilenced, but the problem is still present: trying to run Fish in KBN v K endgame (4-men) with 5-men TBs has no TB hits from Stock.

**Comment by rooklift** [comment]
Hmm. Can you enable logging with the menu `Dev --> Logging` and turn on everything in the submenu. Then try it and post the log here please.

**Comment by AndrewTCEC** [comment]
Here:

**Comment by AndrewTCEC** [comment]
Here: https://drive.google.com/file/d/19QDx0RzrSJOQNw7-c0yuIYQDmofMhvtn/view?usp=sharing, https://drive.google.com/file/d/1Br_jzaK1ugxnUFK-y2pllQ92UEHd_nMA/view?usp=sharing, https://drive.google.com/file/d/1Y2T91Cq4TtlFe88k19PNaT5S7d-7l_ND/view?usp=sharing

**Comment by AndrewTCEC** [comment]
In a nutshell: If you set a SF NetPath with a Cyrillic folder, it crashes.
If you set a TB Path with a Cyrillic folder, it's reading fails and SF continues without TBs.
If there is no Cyrillic in Path, SF runs perfectly fine.

**Comment by rooklift** [comment]
By netpath you mean using a Stockfish EvalFile?

What happens if (as before) you try testing in Stockfish directly? Double-click on stockfish.exe or whatever it's called, and do the following:

```
setoption name EvalFile value PATH
go infinite
```

(Replacing PATH with the exact path to the network file.)

**Comment by rooklift** [comment]
Actually it's not working for me either, even on boring ASCII paths. Not sure why, I'll look into it.

**Comment by AndrewTCEC** [comment]
> By netpath you mean using a Stockfish EvalFile?
> 
Yes, I do. Btw, I forgot to tell you, that the default setting of the integrated net looks to be fine. 

I managed to test it out and... same thing as Leela on the net! Really?

**Comment by rooklift** [comment]
Where did you get the net? I've asked on the Stockfish Discord and many of the nets available at <https://tests.stockfishchess.org/nns> are incompatible with SF13.

**Comment by AndrewTCEC** [comment]
I downloaded there, but I downloaded the actual default net just like it is delivered in abrok.eu builds. 
Net in a path with Cyrillic folder included is picked, but trying to 'go infinite' the application just closes. Path with no Cyrillic works perfectly. 
TBs are a bit different: Cyrillic path shows 0 TBs found, non-Cyrillic works fine. 
That is what I got.

**Comment by rooklift** [comment]
Looks identical to Leela's problem... if this happens even when opening SF in console then it's not a Nibbler issue.

**Comment by AndrewTCEC** [comment]
But then I wonder how Arena handles this like it's nothing... Tried it within the console of Arena with both Leela and Fish, everything runs fine! Some kind of "Translator" is used or what?

**Comment by rooklift** [comment]
That's curious and I have no idea.

Stockfish has an issue about this now: <https://github.com/official-stockfish/Stockfish/issues/3424>
